server/src/models/audit/audit.controller.ts

```diff
@@ -13,6 +13,11 @@ export class AuditController {
   @Auth(Actions.READ, [Features.Security])
   @Get()
   getAudit(): Promise<Audit[]> {
-    return this.service.getAudits({});
+    return this.service.getAudits({
+      orderBy: {
+        createdAt: 'desc',
+      },
+      take: 100,
```

Comment:
I think you should add a minor hint in the UI that states:

```
only the last 100 audit log entries are displayed
```

I would do it after the table with something like this:

![image](https://user-images.githubusercontent.com/11084692/198185915-3cdf38bf-9586-4ba6-a91b-56a7580c495e.png)